### PR TITLE
extend ROUND to lists and GTSs

### PR DESCRIPTION
**Pull request by randomboolean**

### CODE REVIEW

File: warp10/src/main/java/io/warp10/script/WarpScriptLib.java [hunk 1]
```diff
@@ -2799,7 +2798,7 @@ public class WarpScriptLib {
     addNamedWarpScriptFunction(new NumericalUnaryFunction(SIGNUM, null, Math::signum));
     addNamedWarpScriptFunction(new NumericalUnaryFunction(FLOOR, null, Math::floor));
     addNamedWarpScriptFunction(new NumericalUnaryFunction(CEIL, null, Math::ceil));
-    addNamedWarpScriptFunction(new ROUND(ROUND));
+    addNamedWarpScriptFunction(new NumericalUnaryFunction(ROUND, null, Math::round));
```
Comment (steven-gueguen) [inline warp10/src/main/java/io/warp10/script/WarpScriptLib.java:2801]: The `ROUND` function will now return a `DOUBLE` instead of a `LONG`

**Review by steven-gueguen — CHANGES_REQUESTED** [review]

**Review by steven-gueguen — APPROVED** [review]

**Review by pi-r-p — CHANGES_REQUESTED** [review]
Is it possible to add a shortcut when the gts is already a LONG one ? (juste clone it)

**Comment by pi-r-p** [comment]
(I didn't look at #1306 first, but it still applies)

**Comment by randomboolean** [comment]
> Is it possible to add a shortcut when the gts is already a LONG one ? (juste clone it)

PTAL

**Review by pi-r-p — APPROVED** [review]

**Pull request merged**